### PR TITLE
pages.pt_BR/common/*: add Brazilian Portuguese translation

### PR DESCRIPTION
**Pull request by rffontenelle**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.pt_BR/common/clamdscan.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Faz uma varredura no diretório atual e lista apenas os arquivos infectados:
+
+`clamdscan --infected`
+
+- Gera um relatório da varredura para um arquivo de registro:
+
+`clamdscan --log {{caminho/para/arquivo_de_registro}}`
+
+- Gera um relatório da varredura para um arquivo de registro:
```
Comment (renie) [inline pages.pt_BR/common/clamdscan.md:22]:
```suggestion
- Move arquivos infectados para um diretório específico:
```

File: pages.pt_BR/common/clamscan.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Faz uma varredura no diretório atual e lista apenas os arquivos infectados:
+
+`clamscan --infected`
+
+- Gera um relatório da varredura para um arquivo de registro:
+
+`clamscan --log {{caminho/para/arquivo_de_registro}}`
+
+- Gera um relatório da varredura para um arquivo de registro:
```
Comment (renie) [inline pages.pt_BR/common/clamscan.md:30]:
```suggestion
- Move arquivos infectados para um diretório específico:
```

File: pages.pt_BR/common/clang++.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+- Mostra avisos comuns, faz depuração de símbolos na saída e otimiza sem afetar a depuração:
+
+`clang++ {{caminho/para/arquivo_fonte.cpp}} -Wall {{-g|--debug}} -Og {{-o|--output}} {{caminho/para/executável_resultante}}`
+
+- Escolhe um padrão da linguagem para o qual deve-se compilar:
+
+`clang++ {{caminho/para/arquivo_fonte.cpp}} -std={{c++20}} {{-o|--output}} {{caminho/para/executável_resultante}}`
+
+- Inclui bibliotecas localidades em um caminho diferente do arquivo fonte:
```
Comment (renie) [inline pages.pt_BR/common/clang++.md:23]:
```suggestion
- Inclui bibliotecas localizadas em um caminho diferente do arquivo fonte:
```

File: pages.pt_BR/common/clang-format.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+# clang-format
+
+> Formata automaticamente código C/C++/Java/JavaScript/Objective-C/Protobuf/C#.
+> Mais informações: <https://clang.llvm.org/docs/ClangFormat.html>.
+
+- Formata um arquivo e exibe o resultado para a `stdout` (saída padrão):
+
+`clang-format {{caminho/para/arquivo}}`
+
+- Formata um arquivo "in-place", ou seja, salvando nele mesmo:
```
Comment (renie) [inline pages.pt_BR/common/clang-format.md:10]: No changes on this part. Just wanted to say I really like these explanations for terms that usually are not translated. Quite useful! Congrats! :heart:

File: pages.pt_BR/common/crontab.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+- Substitui o crontab atual pelo conteúdo do arquivo fornecido:
+
+`crontab {{caminho/para/arquivo}}`
+
+- Exibe uma lista de tarefas cron existentes para o usuário atual:
+
+`crontab -l`
+
+- Remova todos as tarefas de cron do usuário atual:
```
Comment (renie) [inline pages.pt_BR/common/crontab.md:22]:
```suggestion
- Remove todos as tarefas de cron do usuário atual:
```

**Review by renie — CHANGES_REQUESTED** [review]

File: pages.pt_BR/common/clamdscan.md (hunk 1 above)
Comment (rffontenelle) [reply pages.pt_BR/common/clamdscan.md:22]: WTH did I do there? Thanks for the attention!

File: pages.pt_BR/common/clang-format.md (hunk 4 above)
Comment (rffontenelle) [reply pages.pt_BR/common/clang-format.md:10]: Thanks. Sadly I'm not aware of a better translation for in-place.

Comment (renie) [reply pages.pt_BR/common/clang-format.md:10]: I believe your solution is the best one. To keep the term in English, as it is most used, and give it a short explanation.

**Review by renie — APPROVED** [review]
All good! Thanks for all those new commands, @rffontenelle ! :1st_place_medal:

**Pull request merged**